### ISSUE TITLE
fill_in in a huge selectbox selects the wrong element

### ISSUE THREAD

**Issue by PragTob** [elixir-wallaby/wallaby#469]
:wave:

Me again :)

We have a huge select box holding all the currency iso codes we know of. About 180 of them:

<details>

```
[
    :AED,
    :AFN,
    :ALL,
    :AMD,
    :ANG,
    :AOA,
    :ARS,
    :AUD,
    :AWG,
    :AZN,
    :BAM,
    :BBD,
    :BDT,
    :BGN,
    :BHD,
    :BIF,
    :BMD,
    :BND,
    :BOB,
    :BOV,
    :BRL,
    :BSD,
    :BTN,
    :BWP,
    :BYN,
    :BZD,
    :CAD,
    :CDF,
    :CHE,
    :CHF,
    :CHW,
    :CLF,
    :CLP,
    :CNY,
    :COP,
    :COU,
    :CRC,
    :CUC,
    :CUP,
    :CVE,
    :CZK,
    :DJF,
    :DKK,
    :DOP,
    :DZD,
    :EGP,
    :ERN,
    :ETB,
    :EUR,
    :FJD,
    :FKP,
    :GBP,
    :GEL,
    :GHS,
    :GIP,
    :GMD,
    :GNF,
    :GTQ,
    :GYD,
    :HKD,
    :HNL,
    :HRK,
    :HTG,
    :HUF,
    :IDR,
    :ILS,
    :INR,
    :IQD,
    :IRR,
    :ISK,
    :JMD,
    :JOD,
    :JPY,
    :KES,
    :KGS,
    :KHR,
    :KMF,
    :KPW,
    :KRW,
    :KWD,
    :KYD,
    :KZT,
    :LAK,
    :LBP,
    :LKR,
    :LRD,
    :LSL,
    :LYD,
    :MAD,
    :MDL,
    :MGA,
    :MKD,
    :MMK,
    :MNT,
    :MOP,
    :MRU,
    :MUR,
    :MVR,
    :MWK,
    :MXN,
    :MXV,
    :MYR,
    :MZN,
    :NAD,
    :NGN,
    :NIO,
    :NOK,
    :NPR,
    :NZD,
    :OMR,
    :PAB,
    :PEN,
    :PGK,
    :PHP,
    :PKR,
    :PLN,
    :PYG,
    :QAR,
    :RON,
    :RSD,
    :RUB,
    :RWF,
    :SAR,
    :SBD,
    :SCR,
    :SDG,
    :SEK,
    :SGD,
    :SHP,
    :SLL,
    :SOS,
    :SRD,
    :SSP,
    :STN,
    :SVC,
    :SYP,
    :SZL,
    :THB,
    :TJS,
    :TMT,
    :TND,
    :TOP,
    :TRY,
    :TTD,
    :TWD,
    :TZS,
    :UAH,
    :UGX,
    :USD,
    :USN,
    :UYI,
    :UYU,
    :UYW,
    :UZS,
    :VES,
    :VND,
    :VUV,
    :WST,
    :XAF,
    :XAG,
    :XAU,
    :XBA,
    :XBB,
    :XBC,
    :XBD,
    :XCD,
    :XDR,
    :XOF,
    :XPD,
    :XPF,
    :XPT,
    :XSU,
    :XTS,
    :XUA,
    :XXX,
    :YER,
    :ZAR,
    :ZMW,
    :ZWL
  ]
```
</details>

complete form HTML:

<details>

```
<form accept-charset="UTF-8" action="/admin/currencies" class="test__currency__form" method="post"><input name="_csrf_token" type="hidden" value="enN3O140BAlNGBY0EyUGHk8HIw4r7YL"><input name="_utf8" type="hidden" value="✓">
<label for="currency_currency_iso">Currency iso</label><select id="test__currency__currency-field" name="currency[currency_iso]"><option value="AED">AED</option><option value="AFN">AFN</option><option value="ALL">ALL</option><option value="AMD">AMD</option><option value="ANG">ANG</option><option value="AOA">AOA</option><option value="ARS">ARS</option><option value="AUD">AUD</option><option value="AWG">AWG</option><option value="AZN">AZN</option><option value="BAM">BAM</option><option value="BBD">BBD</option><option value="BDT">BDT</option><option value="BGN">BGN</option><option value="BHD">BHD</option><option value="BIF">BIF</option><option value="BMD">BMD</option><option value="BND">BND</option><option value="BOB">BOB</option><option value="BOV">BOV</option><option value="BRL">BRL</option><option value="BSD">BSD</option><option value="BTN">BTN</option><option value="BWP">BWP</option><option value="BYN">BYN</option><option value="BZD">BZD</option><option value="CDF">CDF</option><option value="CHE">CHE</option><option value="CHW">CHW</option><option value="CLF">CLF</option><option value="CLP">CLP</option><option value="CNY">CNY</option><option value="COP">COP</option><option value="COU">COU</option><option value="CRC">CRC</option><option value="CUC">CUC</option><option value="CUP">CUP</option><option value="CVE">CVE</option><option value="CZK">CZK</option><option value="DJF">DJF</option><option value="DKK">DKK</option><option value="DOP">DOP</option><option value="DZD">DZD</option><option value="EGP">EGP</option><option value="ERN">ERN</option><option value="ETB">ETB</option><option value="FJD">FJD</option><option value="FKP">FKP</option><option value="GEL">GEL</option><option value="GHS">GHS</option><option value="GIP">GIP</option><option value="GMD">GMD</option><option value="GNF">GNF</option><option value="GTQ">GTQ</option><option value="GYD">GYD</option><option value="HKD">HKD</option><option value="HNL">HNL</option><option value="HTG">HTG</option><option value="HUF">HUF</option><option value="IDR">IDR</option><option value="ILS">ILS</option><option value="INR">INR</option><option value="IQD">IQD</option><option value="IRR">IRR</option><option value="ISK">ISK</option><option value="JMD">JMD</option><option value="JOD">JOD</option><option value="KES">KES</option><option value="KGS">KGS</option><option value="KHR">KHR</option><option value="KMF">KMF</option><option value="KPW">KPW</option><option value="KWD">KWD</option><option value="KYD">KYD</option><option value="KZT">KZT</option><option value="LAK">LAK</option><option value="LBP">LBP</option><option value="LKR">LKR</option><option value="LRD">LRD</option><option value="LSL">LSL</option><option value="LYD">LYD</option><option value="MAD">MAD</option><option value="MDL">MDL</option><option value="MGA">MGA</option><option value="MKD">MKD</option><option value="MMK">MMK</option><option value="MNT">MNT</option><option value="MOP">MOP</option><option value="MRU">MRU</option><option value="MUR">MUR</option><option value="MVR">MVR</option><option value="MWK">MWK</option><option value="MXN">MXN</option><option value="MXV">MXV</option><option value="MYR">MYR</option><option value="MZN">MZN</option><option value="NAD">NAD</option><option value="NGN">NGN</option><option value="NIO">NIO</option><option value="NOK">NOK</option><option value="NPR">NPR</option><option value="NZD">NZD</option><option value="OMR">OMR</option><option value="PAB">PAB</option><option value="PEN">PEN</option><option value="PGK">PGK</option><option value="PHP">PHP</option><option value="PKR">PKR</option><option value="PYG">PYG</option><option value="QAR">QAR</option><option value="RON">RON</option><option value="RSD">RSD</option><option value="RUB">RUB</option><option value="RWF">RWF</option><option value="SAR">SAR</option><option value="SBD">SBD</option><option value="SCR">SCR</option><option value="SDG">SDG</option><option value="SEK">SEK</option><option value="SGD">SGD</option><option value="SHP">SHP</option><option value="SLL">SLL</option><option value="SOS">SOS</option><option value="SRD">SRD</option><option value="SSP">SSP</option><option value="STN">STN</option><option value="SVC">SVC</option><option value="SYP">SYP</option><option value="SZL">SZL</option><option value="THB">THB</option><option value="TJS">TJS</option><option value="TMT">TMT</option><option value="TND">TND</option><option value="TOP">TOP</option><option value="TRY">TRY</option><option value="TTD">TTD</option><option value="TWD">TWD</option><option value="TZS">TZS</option><option value="UAH">UAH</option><option value="UGX">UGX</option><option value="USN">USN</option><option value="UYI">UYI</option><option value="UYU">UYU</option><option value="UYW">UYW</option><option value="UZS">UZS</option><option value="VES">VES</option><option value="VND">VND</option><option value="VUV">VUV</option><option value="WST">WST</option><option value="XAF">XAF</option><option value="XAG">XAG</option><option value="XAU">XAU</option><option value="XBA">XBA</option><option value="XBB">XBB</option><option value="XBC">XBC</option><option value="XBD">XBD</option><option value="XCD">XCD</option><option value="XDR">XDR</option><option value="XOF">XOF</option><option value="XPD">XPD</option><option value="XPF">XPF</option><option value="XPT">XPT</option><option value="XSU">XSU</option><option value="XTS">XTS</option><option value="XUA">XUA</option><option value="XXX">XXX</option><option value="YER">YER</option><option value="ZAR">ZAR</option><option value="ZMW">ZMW</option><option value="ZWL">ZWL</option></select>
  <div>
<button class="button" type="submit">Save</button>  </div>
</form>
```

</details>

Now, we used to assign which one we used in a test randomly and it would always fail when selecting `"XXX"`. Upon further inspection/selection it just selected `"XUA"` instead, hence the test was failing.

![xxx](https://user-images.githubusercontent.com/606517/65222059-97fb9700-dabe-11e9-85ff-dfa0e2178e68.png)

For all that I can tell the error occurred only with "XXX". I specifically tested "XUA" and "YER". They both work.

So I'm suspecting that this is some type of inaccuracy that occurs in a very long list and maybe only if a lot of entries start with the same letter.

Not sure if this bug is with wallaby or chromedriver but I haven't had the time to investigate more myself (yet).

## Environment

* Wallaby 0.23.0
* Chromedriver configured to use with chromium, Chromedriver 76
* Elixir 1.9.1
* Erlang 22

edit: forgot the code :scream: 

Basically:

```
      @currency_currency_field Query.select("test__currency__currency-field")
 # ...
    session
    |> fill_in(@currency_currency_field, with: valid_iso_code)
    |> click(@save_button)
```

**Comment by mhanberg** [comment]
Have you tried clicking the option instead of `fill_in`? Not sure if it should work any differently.

```elixir
session
|> click(Query.option("XXX"))
|> click(@save_button)
```

**Comment by mhanberg** [comment]
@PragTob did our comments help you solve this?

**Comment by PragTob** [comment]
Ah hey, sorry. I seem to have missed these notifications :sweat_smile: 

I got so money wallaby notifications that it's sometimes hard to see that some them are my own :grin: 

I'll try it out later, thanks!

**Comment by PragTob** [comment]
Works, thank y'all very much! :ok_hand:
